bin/kano-feedback-coredump

```diff
@@ -2,34 +2,49 @@
 
 # kano-feedback-coredump
 #
-# Copyright (C) 2014, 2015 Kano Computing Ltd.
-# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU General Public License v2
-#
-# This script is called everytime a program segfaults
-# It is setup via sysctl kernel.core_pattern by using a pipe symbol
+# Copyright (C) 2014-2019 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
 #
+# This script is called everytime a program segfaults and runs with sudo.
+# It is setup via sysctl kernel.core_pattern by using a pipe symbol.
+
 
 procname=$1
 killsig=$2
 uid=$3
 gid=$4
 unixtime=$5
 
+latest=2
 core_dump_dir="/var/tmp"
 core_dump_file="$core_dump_dir/core-$procname-$killsig-$uid-$gid-$unixtime.dump"
 
+
 function save_core_dump()
 {
-    # save the coredump coming through stdin to a gzipped file
-    cat < /dev/stdin | gzip -c - > $1.gz
+    # Path to the core.dump file to save as a .gz archive.
+    core_dump_file=$1
+
+    # Save the coredump coming through stdin to a gzipped file.
+    cat < /dev/stdin | gzip -c - > $core_dump_file.gz
```

Comment:
Interesting... this might be what visually takes a substantial amount of time when exiting Dashboard - just a note, looks ok to me.